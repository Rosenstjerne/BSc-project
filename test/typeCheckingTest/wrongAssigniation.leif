var bool b;

b = 4;

print(b);