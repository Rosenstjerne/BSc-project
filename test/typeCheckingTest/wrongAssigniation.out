
Error in phase Type Checking, line 5:
    Trying to assign a value of type 'int' to a varaible of type 'bool' is not possible
